backfill/2024-04-26-active_users_aggregates/desktop_query.sql

```diff
@@ -30,8 +31,32 @@ WITH baseline AS (
   FROM
     `moz-fx-data-shared-prod.telemetry_derived.unified_metrics_v1` AS um
   WHERE
-    um.submission_date = @submission_date
+    um.submission_date BETWEEN DATE_SUB(@submission_date, INTERVAL 28 DAY) AND @submission_date
     AND normalized_app_name = 'Firefox Desktop'
+),
+um_dau AS (
+  SELECT
+    submission_date,
+    client_id,
+    (
+      LOGICAL_AND(uri_count > 0)
+      AND LOGICAL_AND(active_hours_sum > 0)
+      AND LOGICAL_AND(days_since_seen = 0)
+    ) AS is_dau
+  FROM baseline
+  WHERE COALESCE(isp, "") <> 'BrowserStack'
+    AND COALESCE(distribution_id, "") <> 'MozillaOnline'
+  GROUP BY
```

Comment:
These should not be filtered out here; instead we should calculate DAU for the app_names "Firefox Desktop BrowserStack" and "Firefox Desktop MozillaOnline".

We should be calculating DAU/WAU/MAU for all products and segments, even when they don’t necessarily count towards the KPI (Klar and BrowserStack for example). That lets us track e.g. DAU for those segments for reporting purposes (e.g. “How did Mobile Browserstack DAU change in 2024 YTD”), and is consistent with how AUA currently works:

```sql
SELECT submission_date,
       SUM(dau) AS BrowserStack_DAU,
  FROM `moz-fx-data-shared-prod.telemetry.active_users_aggregates`
 WHERE app_name IN ('Fenix BrowserStack',  'Firefox iOS BrowserStack', 'Focus Android BrowserStack', 'Focus iOS BrowserStack')
   AND submission_date >= '2024-01-01'
 GROUP BY submission_date
 ORDER BY submission_date
```
Thanks @jmsilverman for catching this!

Reply:
Thank you! updated.